src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocblocktaglocation/Example2.java

```diff
@@ -0,0 +1,28 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocBlockTagLocation">
+      <property name="tags" value="apiNote, implSpec, implNote"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocblocktaglocation;
+
+class Example2 {
+// xdoc section -- start
+/**
+ * <module name="JavadocBlockTagLocation">
+ *   <!-- default tags -->
+ *   <property name="tags" value="author, deprecated, exception, hidden"/>
```

Comment:
This was mistake of example generation.
This supposed to be config, not a content java file
https://checkstyle.sourceforge.io/version/10.12.0/config_javadoc.html#JavadocBlockTagLocation_Examples
Please move it to header file config area.

Reply:
> This was mistake of example generation. This supposed to be config, not a content java file https://checkstyle.sourceforge.io/version/10.12.0/config_javadoc.html#JavadocBlockTagLocation_Examples Please move it to header file config area.

I am not sure how it should look like in examples file. And, i assume it shouldn't be a part of xdocs or java code. 

I assume something like this but not sure how to alter macros. 

```
/*xml
<module name="Checker">
  <module name="TreeWalker">
    <module name="JavadocBlockTagLocation">
      <property name="tags" value="apiNote, implSpec, implNote"/>
    </module>
  </module>
</module>
*/

/* xml
<module name="Checker">
  <module name="TreeWalker">
    <module name="JavadocBlockTagLocation">
      <property name="tags" value="author, deprecated, exception, hidden"/>
      <property name="tags" value="param, provides, return, see, serial"/>
      <property name="tags" value="serialData, serialField, since, throws"/>
      <property name="tags" value="uses, version"/>
      <property name="tags" value="noinspection"/>
    </module>
  </module>
</module>
*/


package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocblocktaglocation;

class Example2 {
// xdoc section -- start
// xdoc section -- end

}
```

Reply:
Should I create a separate examples file?

Reply:
No, what you have now is good.
In separate issue we can think on how to make some java code to show how this config works. If you see easy way now please add Java code to show no violation and violation cases.